products/PurePhone/PurePhoneMain.cpp

```diff
@@ -272,15 +272,15 @@ int main()
                 platform->deinit();
             }
             catch (const std::runtime_error &e) {
-                LOG_FATAL("%s", e.what());
+                LOG_FATAL("Exception during platform deinit: %s", e.what());
```

Comment:
`LOG_ERROR` here

Reply:
I would consider keeping it as FATAL because after this log we have an abort() function

Reply:
Ok, sure